index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Gowani Pump Supply</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
```

Comment:
"Required meta tags" should be at the top of the head, before title. Especially charset because it defines how to read/parse the page.